apps/expo-go/ios/Podfile.lock

```diff
@@ -2395,7 +2395,7 @@ SPEC CHECKSUMS:
   GoogleDataTransport: 54dee9d48d14580407f8f5fbf2f496e92437a2f2
   GoogleMaps: 032f676450ba0779bd8ce16840690915f84e57ac
   GoogleUtilities: 13e2c67ede716b8741c7989e26893d151b2b2084
-  hermes-engine: 469b31e8c8bd22e85263bd3dd16df539f0c9ca04
+  hermes-engine: 78909e01c02136071e1e884efea014c945c154ba
```

Comment:
nit: double check whether this is expected change

Reply:
I've deleted the pods and run again. It's happening for me on main too. Anything else I should try?

Reply:
no! then it's good to go! thanks for the double check.